ntttcp/ntttcp.Config.json

```diff
@@ -0,0 +1,61 @@
+{
+    "NtttcpDefault": {
+        "Iterations": 5,

```

Comment:
The default case had 1 iteration. Did we want to bump it up? or is this a copy paste error?